src/core/dprint.c

```diff
@@ -57,6 +58,24 @@ void km_log_func_set(km_log_f f)
 	_km_log_func = f;
 }
 
+void km_custom_log_func_set(km_custom_log_f f)
+{
+	_km_custom_log_func = f;
+}
+
+// default custom log format handler,
+// expected to be replaced via km_custom_log_func_set
+void km_default_custom_log_func(int syslog_level, const char *mod_name, const char *file_name,
+							    int line, const char *fmt, ...)
+{
+	char *custom_fmt;
+	asprintf(&custom_fmt, "%s %i:%s:%i %s", mod_name, my_pid(), file_name, line, fmt);
+	va_list args;
+    va_start (args, fmt);
+    vsyslog(syslog_level, custom_fmt, args);
+    va_end (args);
+}
```

Comment:
I think we wrote this default function to stay close to our default log format. We may get asked to keep the Kamailio default format, let's see how that goes...